SIPS/sip-20.md

```diff
@@ -111,6 +112,35 @@ The caveats MUST NOT have duplicate objects with the same `url` properties.
 
 The `url` MUST start with `wss://` which is a protocol indicator for secure WebSocket connection.
 
+### RPC Methods
+
+This SIP introduces new RPC method for sending data to the external connections.
+
+#### snap_sendData
+
+Snap can use `snap_sendData` RPC method to send data specified within request params.
+The proposed RPC method `snap_sendData` SHOULD be a restricted RPC method requiring user consent before usage via the permission system. The RPC method SHOULD only be available to Snaps.
+Snap MUST specify destination which is URL identifier of an external connection (e.g. websocket). Destination MUST be exact match of some of the URLs defined in manifest's permission caveats within `endowment:external-data`.
```

Comment:
In theory the same URL can be used twice, in which case it would be ambiguous which one to use.